precompile/bls12-381/src/lib.rs

```diff
@@ -18,25 +18,42 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+mod bls;
+
 // core
 use core::marker::PhantomData;
+// bls
+use bls::{hash_to_curve_g2, PublicKey, Signature};
 // moonbeam
 use precompile_utils::prelude::*;
 // substrate
 use sp_std::prelude::*;
 
+pub(crate) const BLS_ESTIMATED_COST: u64 = 100_000;
```

Comment:
Let's bump this value bigger, the currently EthBlockGasLimit is 10_000_000 gas. Let change this to the half of the MAX to avoid other big-gas transaction being package the same block.

Reply:
Let me fix it. Give me a sec.

Reply:
No, just keep it, If the pangoro test passed, I will benchmark the cost.

Reply:
Okay. Make sense to me.